### PR TITLE
remove duplicated why

### PR DESCRIPTION
**Pull request by bobjiang**
there is a duplicated why in this sentense.

### CODE REVIEW

File: docs/dev/zksync-v2/contracts.md [hunk 1]
```diff
@@ -4,14 +4,14 @@ To maintain the same security as L1, the zkSync operator must publish the contra
 
 While deploying contracts for the first time may be relatively expensive, factories, which deploy contracts with the same code multiple times, can have huge savings compared to L1.
 
-All these specifics make the process of deploying smart contracts on zkEVM comply with the major rule: _The operator should know the code of the contract before it is deployed_. That means that deploying contracts is only possible by the means of `EIP712` transactions with the `factory_deps` field containing the supplied bytecode. More on EIP712 transactions [here](../../api/api.md#eip712).
+All these specifics make the process of deploying smart contracts on zkEVM comply with the major rule: _The operator should know the code of the contract before it is deployed_. That means that deploying contract is only possible by the means of `EIP712` transactions with the `factory_deps` field containing the supplied bytecode. More on EIP712 transactions [here](../../api/api.md#eip712).
```
Comment (StanislavBreadless) [inline docs/dev/zksync-v2/contracts.md:7]: `deploying contract` sounds weird to me. I guess that `deploying contracts` is better here.

Comment (bobjiang) [reply docs/dev/zksync-v2/contracts.md:7]: get you, removed this change.

Comment (StanislavBreadless) [reply docs/dev/zksync-v2/contracts.md:7]: You should update it here as well (under the comment). 

After that is done, I can merge it

File: docs/dev/zksync-v2/contracts.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ To maintain the same security as L1, the zkSync operator must publish the contra
...
```
Comment (StanislavBreadless) [inline docs/dev/zksync-v2/contracts.md:7]:
```suggestion
All these specifics make the process of deploying smart contracts on zkEVM comply with the major rule: _The operator should know the code of the contract before it is deployed_. That means that deploying contracts is only possible by the means of `EIP712` transactions with the `factory_deps` field containing the supplied bytecode. More on EIP712 transactions [here](../../api/api.md#eip712).
```

**Review by StanislavBreadless — APPROVED** [review]

**Pull request merged**